gateway/1.6/modules/ROOT/pages/policies-included-external-processing.adoc

```diff
@@ -0,0 +1,149 @@
+= External Processing Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:keywords: external processing, processing, flex gateway, gateway, policy
+
+
+[width="100%", cols="5,15"]
+|===
+>s| Policy Name | External Processing
+>s|Summary | Sends the incoming HTTP requests or the outgoing HTTP responses requests to an external gRPC service for additional processing
+>s|Category | Transformation
+>s|First Flex Gateway version available | v1.6.0
+.1+>.^s| Returned Status Codes
+| 403 - IP is rejected
+|===
+
+== Summary
+
+The External Processing policy sends the incoming HTTP request or outgoing HTTP response to an external gRPC service to modify the headers, trailers, body, or any combination of the three. You can also configure this policy to process either the request, response, or both.
+
+Additionally, the external processing service can request different parts of the HTTP request or response after the initial gRPC request has been sent by Flex Gateway to the external processing server. This enables you to send the minimum amount of data necessary. For example, Flex Gateway can initially send only the HTTP requests headers. Then, after some processing, the external processor can request the HTTP request body or trailers from Flex Gateway if necessary. 
+
+The parameters you set for sending the HTTP headers, trailers, and body only configure what is initially sent. The external processing service has access to all parts of the HTTP request or response regardless of what is configured. Configure the parameters to send all the information the processing service needs for every incoming HTTP request or response. For example, if the processing service needs the headers for every request but only needs the body for some of the requests depending on the information found in the header, only configure the parameters to send the headers.
+
+The External Processing policy is an extension of the External Processing filter native to Envoy. To learn more about the External Processing filter, see https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/ext_proc/v3/ext_proc.proto[External Processing Filter^]. Not all External Processing filter parameters are avialible for the External Processing policy. To find what parameters are configurable, see <<configuring-policy-parameters, Configuring Policy Parameters>>. 
```

Comment:
I think that you can shorten the reference to: "To learn more, see..."

Reply:
I think I would prefer to keep the first sentence in, if that's ok.

Reply:
Fine